cupy/testing/helper.py

```diff
@@ -137,6 +137,10 @@ def _contains_signed_and_unsigned(kw):
 
 def _make_decorator(check_func, name, type_check, accept_error, sp_name=None,
```

Comment:
tests use `accept_error` as kwarg, should we force the arguments to be kwarg only?

Reply:
`_make_decorator` is an internal function, and there are only a few callers. I don't think there's any reason to make it keyword-only.

Reply:
If you think that increases readability, I won't object that, though.

Reply:
with the reduced number of cases and the unlikely addition of more, it is ok to leave it as-is. I was suggesting it only to enforce using the same style in the code base, but as you said probably it won't be used in other places.